pkg/build/graph/analysis/bc.go

```diff
@@ -162,6 +162,91 @@ func FindCircularBuilds(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
 	return markers
 }
 
+// multiBCStartBuildSuggestion builds the `oc start-build` suggestion string with multiple build configs
+func multiBCStartBuildSuggestion(bcNodes []*buildgraph.BuildConfigNode) string {
+	var ret string
+	if len(bcNodes) > 1 {
+		ret = "Run one of the following commands: "
+	}
+	for i, bcNode := range bcNodes {
+		// use of f.ResourceName(bcNode) will produce a string like  oc start-build BuildConfig|example/ruby-hello-world
+		ret = ret + fmt.Sprintf("oc start-build %s", bcNode.BuildConfig.GetName())
+		if i < (len(bcNodes) - 1) {
+			ret = ret + ", "
```

Comment:
> This makes the line not copy-paste-able into the shell... do we care? Do we want each suggestion on it's own line?

I think this is better.  Otherwise I'll get tons of lines.